src/main/scripts/setup/mesos_marathon_setup.py

```diff
@@ -0,0 +1,215 @@
+# Zookeeper to keep track of the current leader of the master servers.
+# The Mesos layer, built on top of this, will provide distributed synchronization and resource handling.
+# It is responsible for managing the cluster.
+# Marathon, the cluster's distributed init system, is used to schedule tasks and hand work to the slave servers.
+
+import os
+import argparse
+import setup_helpers
+import ConfigParser
+from shell_command import shell_call
+import ntpath
+from fabric.api import *
+
+parser = argparse.ArgumentParser(description='Mesos Marathon setup script')
+# 'default=3' fulfills the Apache Mesos recommendation of having at least three masters for a production environment.
+parser.add_argument('--config_file', '-f', type=str, default=os.getcwd() + "/setup_config.ini", help='Setup configuration file')
+
+parser.add_argument('--local_work_dir', '-l', type=str, default=os.environ['HOME'], help='Script will copy all downloaded/output files in this directory')
```

Comment:
If not specified you can either choose current directory or put it in a hidden directory in home 
i.e. $HOME/.hydra